src/main/java/com/tripply/booking/model/request/RoomBookingRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.tripply.booking.model.request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class RoomBookingRequest {
+    @NotNull(message = "It can't be null")
+    @NotBlank(message = "It can't be blank")
+    private String checkInTime;
+    @NotNull(message = "It can't be null")
+    @NotBlank(message = "It can't be blank")
+    private String checkOutTime;
+    @NotNull(message = "It can't be null")
```

Comment:
Use a single custom annotation instead of two different ones. Create one custom annotation to handle your condition effectively.

Reply:
I just checked that @NotBlank annotation is equal to the combination of not null check and not blank check. So there is no need to create custom annotation.